app/src/main/res/layout/detailed_recipe_fragment.xml

```diff
@@ -0,0 +1,95 @@
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/detailed_recipe_image"
+            android:layout_width="1000dp"
+            android:layout_height="300dp"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentTop="true"
+            android:adjustViewBounds="true"
+            android:scaleType="centerCrop" />
+
+        <TextView
+            android:id="@+id/ingredients_required"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/detailed_recipe_image"
+            android:layout_margin="8dp"
+            android:text="Ingredients required"
+            android:textColor="@android:color/black"
+            android:textSize="25sp" />
+
+        <TextView
+            android:id="@+id/detailed_recipe_detail"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/ingredients_required"
+            android:layout_margin="8dp" />
+
+        <TextView
+            android:id="@+id/howcook"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/detailed_recipe_detail"
+            android:layout_margin="8dp"
+            android:text="How to cook:"
+            android:textColor="@android:color/black"
+            android:textSize="23sp" />
+
+        <TextView
+            android:id="@+id/how_to_cook"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/howcook"
+            android:layout_marginEnd="8dp"
+            android:layout_marginStart="8dp" />
+
+        <TextView
+            android:id="@+id/numb_of_recipe"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/how_to_cook"
+            android:layout_margin="10dp"
+            android:text="Chosen recipes"
+            android:textColor="@android:color/black"
+            android:textSize="16sp" />
+
+        <android.support.design.widget.FloatingActionButton
+            android:id="@+id/floatingActionButton3"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignEnd="@+id/numb_of_recipe"
+            android:layout_alignParentBottom="true"
+            android:layout_margin="13dp"
+            android:layout_marginEnd="24dp"
+            android:clickable="true"
+            android:paddingStart="2dp"
+            app:elevation="0dp"
+            app:fabSize="mini"
+            app:srcCompat="@drawable/ic_plus_one_black" />
+
+        <android.support.design.widget.FloatingActionButton
```

Comment:
are you sure you want to use floating action button? according to android UI guidelines this component is for completely different purposes